app/src/main/java/mozilla/lockbox/store/RouteStore.kt

```diff
@@ -25,15 +33,29 @@ class RouteStore(
         val shared = RouteStore()
     }
 
+    internal val compositeDisposable = CompositeDisposable()
+
+    private val onboarding: Observable<Boolean> = BehaviorRelay.createDefault(false)
+
     val routes: Observable<RouteAction> = ReplaySubject.createWithSize(1)
 
     init {
         dispatcher.register
             .filterByType(RouteAction::class.java)
+            .debug("dispatched route actions")
```

Comment:
looks like you committed a few extra `.debug`s... generally, these aren't good to leave in production code!!

Reply:
shoot! good call